py/selenium/webdriver/chromium/webdriver.py

```diff
@@ -31,10 +30,10 @@ class ChromiumDriver(RemoteWebDriver):
 
     def __init__(
         self,
-        browser_name: Optional[str] = None,
-        vendor_prefix: Optional[str] = None,
-        options: ArgOptions = ArgOptions(),
-        service: Optional[Service] = None,
+        browser_name: str,
+        vendor_prefix: str,
+        options: ChromiumOptions = ChromiumOptions(),
+        service: ChromiumService = ChromiumService(),
```

Comment:
`options` and `service` are always supposed to be `ChromiumOptions` and `ChromiumService` respectively.  
In fact, they can never be `None` according to the code in the `__init__`.  

This doesn't directly impact `chrome.webdriver.WebDriver.__init__` nor `edge.webdriver.WebDriver.__init__` which both allow the arguments to be `None`; both create instances if the user doesn't pass it in.